yarn.lock

```diff
@@ -10243,10 +10360,11 @@ worker-farm@^1.5.2:
     errno "~0.1.7"
 
 "workerjs@github:jasonLaster/workerjs":
-  version "0.1.2"
-  resolved "https://codeload.github.com/jasonLaster/workerjs/tar.gz/8c7df3e465b1d096d5134c648287d4a6f9c3e4ea"
```

Comment:
I don't understand fully how the dependencies work, but what I saw it that the workerjs package installed in node_modules wasn't what this URL pointed too. And bumping the version make it work properly.

My theory is that we had a 0.1.2 in our local cache, and yarn was just taking this locally. Locally the error was a babel version mismatch.
On CI, we had the right package, but bad dependencies in yarn.lock because of this, and the error message was `can't find @babel/register`.